plugins/header_rewrite/value.cc

```diff
@@ -55,6 +55,7 @@ Value::set_value(const std::string &val)
 
           if (parser.parse_line(_value)) {
             tcond_val->initialize(parser);
+            require_resources(tcond_val->get_resource_ids());
```

Comment:
why does Value need the resource ids? doesn't this get defined when initialize() is called